src/OpenIddict.Models/OpenIddictToken.cs

```diff
@@ -73,7 +73,19 @@ public class OpenIddictToken<TKey, TApplication, TAuthorization> where TKey : IE
         /// Gets or sets the unique identifier
         /// associated with the current token.
         /// </summary>
-        public virtual TKey Id { get; set; }
+        public virtual TKey TokenId { get; set; }
+
+        /// <summary>
+        /// Gets or sets the unique identifier
+        /// associated with the current application.
+        /// </summary>
+        public virtual TKey ApplicationId { get; set; }
```

Comment:
You can't do that. Read the warning here: https://github.com/openiddict/openiddict-core/blob/dev/src/OpenIddict.EntityFrameworkCore/OpenIddictExtensions.cs#L216-L218.